examples/firecrawl_custom_config_example.ts

```diff
@@ -0,0 +1,183 @@
+/**
```

Comment:
can we change this to just use langgraph's `createReactAgent`? it does exactly the same thing under the hood

Reply:
Done